src/test/java/com/marizueva/laboratory/hw2/ex2/LoginTestWithSoftAssertions.java

```diff
@@ -0,0 +1,265 @@
+package com.marizueva.laboratory.hw2.ex2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class LoginTestWithSoftAssertions {
+    SoftAssert softAssert;
+
+    private WebDriver driver;
+    private  final String baseUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    private final String login = "Roman";
+    private final String password = "Jdi1234";
```

Comment:
Looks like that this is same variables as in `src/test/java/com/marizueva/laboratory/hw2/ex1/LoginTestWithHardAssertions.java` class. I suppose that its could be extracted to some `Base` class

Reply:
done